frontend/src/screens/channels/Channels.tsx

```diff
@@ -318,15 +318,11 @@ export default function Channels() {
                   {new Intl.NumberFormat().format(balances.onchain.spendable)}{" "}
                   sats
                   {balances &&
-                    (balances.onchain.spendable !== balances.onchain.total ||
-                      balances.onchain.pendingBalancesFromChannelClosures >
-                        0) && (
+                    balances.onchain.spendable !== balances.onchain.total && (
                       <p className="text-xs text-muted-foreground animate-pulse">
                         +
                         {new Intl.NumberFormat().format(
-                          balances.onchain.total -
-                            balances.onchain.spendable +
-                            balances.onchain.pendingBalancesFromChannelClosures
+                          balances.onchain.total - balances.onchain.spendable
```

Comment:
The docs for `pending_balances_from_channel_closures` say:

> ...depending on the sync status of the wallets, swept balances listed here might or might not already be accounted for in [total_onchain_balance_sats](https://docs.rs/ldk-node/latest/ldk_node/struct.BalanceDetails.html#structfield.total_onchain_balance_sats)

I searched if there is any method which provides the sync status so it can be used in the get balances method but couldn't find any, do you know of such method?

Not sure if it's worth the effort, but from this change, my understanding is that if LDK and BDK are not in sync, then the user might not even see a dialog/incoming message after a channel is closed and is missing from the list like they do today for sometime (till they sync again)

Reply:
Nice find!

I think LDK and BDK are in sync, but it means there is some possible overlap in funds for a short period of time.

Because of this possibility that for a short period of time you will see funds from closed channels in two places, we now never sum the pending balances with the total balance, to avoid showing duplicated funds.